appspec.yml

```diff
@@ -3,11 +3,11 @@ os: linux
 files:
   - source: out/artifacts/java_http_server_jar/java-http-server.jar
     destination: /opt
-  - source: deploy_server.sh
+  - source: /scripts/start_server.sh
     destination: /opt
   - source: /Logs
     destination: /opt
 hooks:
-  ApplicationStart:
-    - location: deploy_server.sh
+  AfterInstall:
+    - location: ../../../../../start_server.sh
```

Comment:
Any luck on the absolute path here?

Reply:
Just put these two changes on a separate branch - going to test and then will make a PR once it's working!